master/getting-started/kubernetes/installation/hosted/canal/index.md

```diff
@@ -20,16 +20,13 @@ system.
 
 ## Before you begin 
 
-- Ensure that your cluster meets the [requirements](../../../reqs/system).
-  An easy way to create a cluster which meets these requirements is by 
+- Ensure that your cluster meets the {{site.prodname}} [System requirements](../../../reqs/system).
+  An easy way to create a cluster which meets the requirements is by 
   following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
-
-- If you are upgrading from the Kubernetes [1.6](https://github.com/projectcalico/canal/blob/master/k8s-install/README.md#for-kubernetes-16)
-  or [1.5](https://github.com/projectcalico/canal/blob/master/k8s-install/README.md#kubernetes-15)
-  manifests from the (deprecated) Canal repo to the manifests here it is
-  necessary to [migrate your {{site.prodname}} configuration data](https://github.com/projectcalico/calico/blob/master/upgrade/v2.5/README.md) 
-  before upgrading. Otherwise, your cluster may lose connectivity after 
-  the upgrade.
+  
+- The Kubernetes controller manager must be started with
+  `--cluster-cidr=10.244.0.0/16` and `--allocate-node-cidrs=true`. If using kubeadm, specifying 
+  `--pod-network-cidr=10.244.0.0/16` will ensure the above flags are set.
```

Comment:
Since this is one line now, it should probably say "ensure these flags are set" rather than "the above flags"